### PR TITLE
feat(web): initialize web

### PR DESCRIPTION
**Pull request by issmail-basel**

### CODE REVIEW

File: web/codegen.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        Cursor: string
+        URL: string
+        Lang: string
+        TranslatedString: '{ [lang in string]?: string } | null'
+        PluginID: string
+        PluginExtensionID: string
+        PropertySchemaID: string
+        PropertySchemaGroupID: string
+        PropertySchemaFieldID: string
+        DatasetSchemaFieldID: string
```
Comment (rot1024) [inline web/codegen.yml:28]:
```suggestion
        DateTime: Date
        FileSize: number
        ID: string
        Cursor: string
        URL: string
        Lang: string
```

File: web/codegen.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+overwrite: true
+schema: http://localhost:8080/api/graphql
```
Comment (rot1024) [inline web/codegen.yml:2]:
```suggestion
schema: ../server/schema.graphql
```

The backend is in the same repository, so you can retrieve the schema locally!

File: web/src/components/atoms/NotFound/index.tsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { useIntl } from "react-intl";
```
Comment (rot1024) [inline web/src/components/atoms/NotFound/index.tsx:2]: Can you introduce react-i18nnext instead of react-intl?

File: web/src/main.tsx [hunk 4]
```diff
@@ -3,10 +3,13 @@ import ReactDOM from "react-dom";
...
+import loadConfig from "./config";
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById("root")
-);
+loadConfig().finally(() => {
+  ReactDOM.render(
```
Comment (rot1024) [inline web/src/main.tsx:9]: ReactDOM.render is deprecated in React v18. Use `ReactDOM.createRoot`

**Review by rot1024 — CHANGES_REQUESTED** [review]
Generally OK, but

- Use react-i18nnext instead of react-intl
- Use `ReactDOM.createRoot`

File: web/src/components/atoms/NotFound/index.tsx (hunk 3 above)
Comment (issmail-basel) [reply web/src/components/atoms/NotFound/index.tsx]: I removed `react-intl` for now.
Will add `react-i18nnext` it in another issue

**Review by rot1024 — APPROVED** [review]

**Pull request merged**